packages/shared-ux/button_toolbar/src/buttons/icon_button_group/icon_button_group.styles.ts

```diff
@@ -14,11 +14,13 @@ export const IconButtonGroupStyles = ({ euiTheme }: UseEuiTheme) => {
       '&.euiButtonGroupButton': {
         backgroundColor: euiTheme.colors.emptyShade,
         border: `${euiTheme.border.thin} !important`,
+        borderRight: 'none !important',
```

Comment:
This fixes the old "double border" between each of the icon buttons in the `IconButtonGroup`: